config/lab_manager.example.yml

```diff
@@ -1,12 +1,11 @@
 default: &default
-  log_level: <%= Logger::WARN %>
+  log_level: <%= Logger::DEBUG %>
```

Comment:
Logging is a potentially expensive operation. Please set the level at least to warn.